extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/vertx/BufferSerializer.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2011-2019 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+package io.quarkus.jackson.runtime.vertx;
+
+import static io.vertx.core.json.impl.JsonUtil.BASE64_ENCODER;
```

Comment:
Because it's in "impl", I would not use that, but create another instance.

Reply:
Done